main.py

```diff
@@ -202,15 +220,33 @@ def clusters(methods=["GET"]):
         sw_lng = float(request.values['sw_lng'])
         start_date = datetime.date.fromtimestamp(int(request.values['start_date']))
         end_date = datetime.date.fromtimestamp(int(request.values['end_date']))
-        fatal = int(request.values['show_fatal'])
-        severe = int(request.values['show_severe'])
-        light = int(request.values['show_light'])
-        inaccurate = int(request.values['show_inaccurate'])
+        fatal = bool(request.values['show_fatal'])
```

Comment:
This can be done easily. 
Yet the number of lines would stay the same, only as function calls.
Is it better practice to use a function even so?